sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -1559,7 +1559,8 @@ object PushPredicateThroughJoin extends Rule[LogicalPlan] with PredicateHelper {
 /**
  * This rule optimizes Limit operators by:
  * 1. Eliminate [[Limit]] operators if it's child max row <= limit.
- * 2. Combines two adjacent [[Limit]] operators into one, merging the
+ * 2. Eliminate [[GlobalLimit]] operators if it's child max row <= limit.
```

Comment:
nit: how about merging `1.` and `2.` like `Eliminate [[Limit]]/[[GlobalLimit]] ...`?

Reply:
fixed.